src/components/ImageDepViewer.js

```diff
@@ -56,12 +56,16 @@ export const ImageDepViewer = ({ packageLink }) => {
           ])
         ]),
         tbody(
-          _.map(({ name, version }) => {
-            return tr({ key: name }, [
-              td({ style: { paddingRight: '1rem' } }, [name]),
-              td([version])
-            ])
-          }, _.filter({ tool: selectedTool }, packages))
```

Comment:
Thanks for making the code below `flow`y. It reads more easily despite the added function :)